runtime/doc/ui.txt

```diff
@@ -23,40 +23,37 @@ screen grid with a size of width × height cells. This is typically done by an
 embedder at startup (see |ui-startup|), but UIs can also connect to a running
 Nvim instance and invoke nvim_ui_attach(). The `options` parameter is a map
 with these (optional) keys:
+
 							*ui-rgb*
-	`rgb`			Decides the color format.
-				true:	(default) 24-bit RGB colors
-				false:	Terminal colors (8-bit, max 256)
+- `rgb`			Decides the color format.
+			- true:	(default) 24-bit RGB colors
+			- false:	Terminal colors (8-bit, max 256)
 							*ui-override*
-	`override`		Decides how UI capabilities are resolved.
-				true:	Enable requested UI capabilities, even
-					if not supported by all connected UIs
-					(including |TUI|).
-				false:	(default) Disable UI capabilities not
-					supported by all connected UIs
-					(including TUI).
+- `override`		Decides how UI capabilities are resolved.
+			- true:	Enable requested UI capabilities, even if not
+			  supported by all connected UIs (including |TUI|).
+			- false: (default) Disable UI capabilities not
+			  supported by all connected UIs (including TUI).
 							*ui-ext-options*
-	`ext_cmdline`		Externalize the cmdline. |ui-cmdline|
-	`ext_hlstate`		Detailed highlight state. |ui-hlstate|
-				Sets `ext_linegrid` implicitly.
-	`ext_linegrid`		Line-based grid events. |ui-linegrid|
-				Deactivates |ui-grid-old| implicitly.
-	`ext_messages`		Externalize messages. |ui-messages|
-				Sets `ext_linegrid` and `ext_cmdline` implicitly.
-	`ext_multigrid`		Per-window grid events. |ui-multigrid|
-				Sets `ext_linegrid` implicitly.
-	`ext_popupmenu`		Externalize |popupmenu-completion| and
-				'wildmenu'. |ui-popupmenu|
-	`ext_tabline`		Externalize the tabline. |ui-tabline|
-	`ext_termcolors`	Use external default colors.
-	`term_name`		Sets the name of the terminal 'term'.
-	`term_colors`		Sets the number of supported colors 't_Co'.
-	`term_background`	Sets the default value of 'background'.
-	`stdin_fd`		Read buffer from `fd` as if it was a stdin pipe
-				This option can only used by |--embed| ui,
-				see |ui-startup-stdin|.
-
-
+- `ext_cmdline`		Externalize the cmdline. |ui-cmdline|
+- `ext_hlstate`		Detailed highlight state. |ui-hlstate|
```

Comment:
Use lists to list things... otherwise we are stuck with "hardwrap" / preformatted format.